cafebabel/templates/articles/macros.html

```diff
@@ -34,3 +34,20 @@
     <ul id=tags-suggestions class=inactive></ul>
   </div>
 {%- endmacro %}
+
+{%- macro image_upload(article) -%}
```

Comment:
I didn't know how to get `article` dynamic. In tag/edit, I would have liked to use this macros while replacing `article` by `tag`. 
I tried something like:
`<output><img src={{ {{ article}}.image_url }} alt="Image for {{ {{ article}}.name }}">`
but it obviously doesn't work 👎 
any suggestion @vinyll ?